eventbridge-pipes-sqs-to-redshift/README.md

```diff
@@ -0,0 +1,146 @@
+
+## Getting Started
+The entire solution is built using AWS Cloud Development Kit (CDK) in Python. The instructions below shows how solution works, deployment instructions, testing steps and cleanup.
+
+### How it works
+The solution will create two Amazon SQS queues, Amazon Redshift Cluster, and an EventBridge Pipes to connect it all.
+- EventBridge pipes internally create consumers for SQS queue and polls for new messages from source queue 
+- EventBridge pipes transform the SQS message to create insert queries from data within message and then send the transformed messages to a target SQS queue. 
+- EventBridge pipes then run Redshift data API queries to the insert data to Amazon Redshift cluster `pipestarget` in `private` database in `messages` table and add the SQS messages
+
+For more informations: [Amazon EventBridge Pipes with SQS ](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes-sqs.html) and  [Amazon Redshift Data API ](https://docs.aws.amazon.com/redshift/latest/mgmt/data-api-calling-event-bridge.html)
+
+### Architecture
+![Alt text](SQS_Redshift_Pipe.jpg)
+
+### Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+
+2. Change directory to the pattern directory:
+    ```
+    cd eventbridge-pipes-sqs-to-redshift
+    ```
+
+3. Once the repository is cloned open terminal on project directory and create a virtualenv on MacOS and Linux:
+    ```
+    python3 -m venv .venv
+    ```
+
+4. After the init process completes and the virtualenv is created, you can use the following step to activate your virtualenv.
+    ```
+    source .venv/bin/activate
+    ```
+
+    In Windows platform, you would activate the virtualenv like this:
+    ```
+    .venv\Scripts\activate.bat
+    ```
+
+5. Once the virtualenv is activated, you can install the required dependencies.
+    ```
+    pip install -r requirements.txt
+    ```
+
+6. At this point you can now synthesize the CloudFormation template for this code.
+    ```
+    cdk synth
+    ```
+
+7. Once the synthesize is successful deploy this stack
+    ```
+    cdk deploy
+    ```
+
+## Testing Steps
+For testing you will be using AWS CLI commands to send a message source SQS queue and Redshift cluster data API queries to read data from Redshift cluster.
+
+1. Send a test message to source SQS queue, run the below CLI command and replace `--queue-url` with source SQS queue url from CDK deployment output  
+    ```
+    aws sqs send-message --queue-url <Source SQS Queue URL:> --message-body "Message To Redshift via EventBridge Pipe."
+    ```
+
+    Output sample response:
+    ```
+    {
+        "MD5OfMessageBody": "c905630d3f30e240e0f467960bf80dd5",
+        "MessageId": "696ae5f3-5581-4283-9cc0-abd599f2f1f3"
+    }
+    ```
+
+2. To to fetch data from Redshift cluster, run the blew Redshift cluster data API queries
```

Comment:
typos in statement. additional 'to' and 'belw' instead of 'below'

Reply:
Thanks. Fixed now